assets/js/util.js

```diff
@@ -242,8 +243,29 @@ function synchronizeTextareaHeights() {
     $('#translatedText').css('height', originalTextScrollHeight + 'px');
 }
 
-/*:: export {synchronizeTextareaHeights, modeEnabled, ajaxSend, ajaxComplete, filterLangList, onlyUnique,
-    SPACE_KEY_CODE, ENTER_KEY_CODE, HTTP_OK_CODE, HTTP_BAD_REQUEST_CODE, XHR_LOADING, XHR_DONE} */
+function ajaxCallForAllModes(methodType, request, endpoint, callbackSuccess, callbackError, requestName) {
+    requestName = $.ajax({
+        url: config.APY_URL + endpoint,
+        beforeSend: ajaxSend,
+        complete: function () {
+            ajaxComplete();
+            requestName = undefined;
```

Comment:
Yep, I realised that now! Um, a workaround could be as mentioned in this [link](https://stackoverflow.com/questions/18178728/passing-a-global-variable-to-a-function). I would make use of `window[requestName]` to achieve the above functionality and pass the request names as a String argument. Inputs?